maxcul.coffee

```diff
@@ -73,9 +73,18 @@ module.exports = (env) ->
             type: "number"
             acronym: "T"
             unit: "°C"
+        },
+        {
```

Comment:
Do we really need this? We extend the HeatingThermostat class of pimatic and this already contains the batty attribute https://github.com/pimatic/pimatic/blob/master/lib/devices.coffee#L541